package.json

```diff
@@ -70,7 +70,8 @@
     "dayjs": "^1.8.13",
     "lodash": "^4.17.11",
     "path": "^0.12.7",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "yarn": "^1.16.0"
```

Comment:
i'm not sure if we need yarn here as module. at my PC yarn installed globally

Reply:
Yeah, not necessary. Just installed to remove the error of npm. Should I uninstall?

Reply:
yes please